src/EditorFeatures/Core/Classification/Semantic/ClassificationUtilities.cs

```diff
@@ -162,7 +163,7 @@ public static List<ITagSpan<IClassificationTag>> Convert(IClassificationTypeMap
             {
                 using (Logger.LogBlock(FunctionId.Tagger_SemanticClassification_TagProducer_ProduceTags, cancellationToken))
                 {
-                    using var _ = ArrayBuilder<ClassifiedSpan>.GetInstance(out var classifiedSpans);
+                    var classifiedSpans = ImmutableSegmentedList.CreateBuilder<ClassifiedSpan>();
```

Comment:
can the builder be pooled as well?

Reply:
Possibly, but I'm not seeing strong evidence of value. If there was measurable benefits from it, we already have `ImmutableSegmentedList<T>.ValueBuilder` where the pool doesn't seem to provide value on top of that.

Reply:
so classification is called *a lot* :)   we originally moved to pooled here off of other collections because of that.  i think there are two separate concerns we have:

1. allocations
2. LOH allocations.

i worry about this addressing '2' while exacerbating 1.